testsuite/tests/backtrace/backtrace_effects_nested.ml

```diff
@@ -1,6 +1,4 @@
 (* TEST
-   * skip
-
```

Comment:
Why delete this? That seems to contradict #191

Reply:
I'll resolve by restoring this and similar, and let you object before merging

Reply:
I think that was just a mistake, these should be skipped.